_includes/_blogcontent/2024-03-06-introducing-scix-ambassador-program.md

```diff
@@ -0,0 +1,37 @@
+
+The Lead Ambassadors will serve a two-year term, during which they will participate in monthly online meetings with their cohort and the NASA SciX team. The estimated time commitment for these activities is between 2 and 8 hours per month, ensuring that ambassadors can balance their responsibilities with their ongoing research and professional commitments.
+
+The NASA SciX Ambassador Program aims to empower ambassadors to effectively promote the platform, improve it through community feedback, gain visibility and recognition for their work, and build a robust online community that connects researchers from diverse fields. Through their efforts, the ambassadors will play a pivotal role in advancing the mission of interdisciplinary research and open science.
+
+### Key Program Highlights
+
+- **Presentations:** Ambassadors will present on NASA SciX twice a year in various formats, such as locally at their institute, virtually at a meeting, or in person at a workshop.
+- **Workshop:** A 2.5-day workshop at the [Center for Astrophysics | Harvard & Smithsonian](https://www.cfa.harvard.edu/) (August 21 - 23, 2024), where ambassadors will connect with the NASA SciX team, learn community engagement skills, and receive training on the NASA SciX platform.
+- **Online Community:** Ambassadors will have access to an online space to share resources, connect with the NASA SciX team, and collaborate with peers worldwide.
+- **Recognition:** Ambassadors will be featured on our SciX website and have opportunities to showcase their work and community engagement on our blogs.
+- **Travel Support:** Up to $2500 for workshop attendance and an additional $2500 to attend a conference of their choice to present on their work and NASA SciX.
+- **Duration:** Ambassadors will serve a two-year term, participating in monthly online meetings with their cohort and the NASA SciX team. The estimated time commitment is 2 - 8 hours per month.
+
+Interested in becoming a NASA SciX Lead Ambassador? Fill out our short application form [here](https://docs.google.com/forms/d/e/1FAIpQLSeMmhfklhEDl7sc2uP9Tk7hSQrSjHiTBwL42FIeuYB5L4dOBA/viewform). We especially encourage early career researchers (including graduate students) and members of underrepresented groups in STEM to apply. We accept international applications as well. 
+
+Lead ambassadors will be instrumental in sharing the benefits and features of NASA SciX with their communities and expanding our overall NASA SciX Ambassador Program. We are actively developing educational and training material for educators and ambassadors to present NASA SciX to their communities and we will advertise these resources as they become available. Join us in this exciting journey to bridge disciplines, enhance scientific communication, and foster a community of innovation through the NASA SciX Ambassador Program!
```

Comment:
Should we include an email address (yours, or `help@scixplorer.org`) for them to direct any question to?